src/m365/spo/commands/hubsite/hubsite-get.ts

```diff
@@ -39,8 +68,25 @@ class SpoHubSiteGetCommand extends SpoCommand {
 
         return request.get(requestOptions);
       })
-      .then((res: any): void => {
-        logger.log(res);
+      .then((res: HubSite): Promise<AssociatedSite[] | void> => {
+        hubSite = res;
+
+        if (args.options.includeAssociatedSites !== true || args.options.output && args.options.output !== 'json') {
+          return Promise.resolve();
+        }
+
+        return spo
+          .getSpoAdminUrl(logger, this.debug)
+          .then((spoAdminUrl: string): Promise<AssociatedSite[] | void> => {
+            return this.getAssociatedSites(spoAdminUrl, logger, args);
+          });
+      })
+      .then((associatedSites: AssociatedSite[] | void): void => {
+        if (associatedSites) {
+          hubSite.AssociatedSites = associatedSites.filter(s => s.SiteId !== args.options.id);
+        }
```

Comment:
lets change the input to something like `associatedSitesCommandOutput: CommandOutput | void`
and then lets do the parsing here like

```
if (associatedSitesCommandOutput) {
          const associatedSites = JSON.parse((associatedSitesCommandOutput as CommandOutput).stdout) as AssociatedSite[];
          hubSite.AssociatedSites = associatedSites.filter(s => s.SiteId !== args.options.id);
}
```